README.md

```diff
@@ -48,6 +48,24 @@ Some of the actions provided by debos to customize and produce images are:
 A full syntax description of all the debos actions can be found at:
 https://godoc.org/github.com/go-debos/debos/actions
 
+### Origins
+
+A couple of actions offered by debos allow access to files on the host
+or the target system. The base location of the files is defined with
+the action's ``origin`` property. This property has the following
+predefined locations:
+
+* ``artifacts``: By default the directory in which debos is invoked.
+
+  It can be set with the argument ``--artifactdir $directory``.
+
+* ``filesystem``: By default ``$scratchdir/root``.
```

Comment:
Filesystem is the path to the filesystem that is currently being build up. (iotw it will change after runing the deploy actions). The location is really an implementation detail